### PR TITLE
Run builds for all branches, not for Pull Requests

### PR DESCRIPTION
**Pull request by scinos**
### Background

TeamCity is configured to run builds only for open Pull Requests that come from a org member or external collaborator.
PRs open by renovate are not run because renovate bot is not a member or colaborator.

### Changes proposed in this Pull Request

* Change TeamCity config to build all branches, not only Pull Requests

### Testing

* Open TeamCity build and verify there are no errors when running build specs DSL.
* The actual change can't be properly testing until it is merged though

### CODE REVIEW

File: .teamcity/settings.kts [hunk 1]
```diff
@@ -335,11 +333,7 @@ object RunAllUnitTests : BuildType({
 
 	triggers {
 		vcs {
-			branchFilter = """
-				+:pull/*
-				+:master
-				+:trunk
-			""".trimIndent()
+			branchFilter = "+:*"
```
Comment (scinos) [inline .teamcity/settings.kts:336]: Generated by TeamCity UI. This means "run this build for all branches"

File: .teamcity/settings.kts [hunk 2]
```diff
@@ -351,16 +345,6 @@ object RunAllUnitTests : BuildType({
 		}
 		perfmon {
 		}
-		pullRequests {
```
Comment (scinos) [inline .teamcity/settings.kts:354]: This plugin was getting a list of open Pull Requests and creating "fake" branches named `pull/<id>`, so other parts of TeamCity (for example, any `branchFilter`) can filter them.

**Review by tyxla — APPROVED** [review]
This LGTM! 🚢  Let's give it a try.

Do you think there will be large overhead added by builds made on branches that never make it to the phase where they're PRs waiting for review? Any concerns that will generally slow down the TC build speed for everyone?

**Comment by scinos** [comment]
> Do you think there will be large overhead added by builds made on branches that never make it to the phase where they're PRs waiting for review? Any concerns that will generally slow down the TC build speed for everyone?

I don't expect it to cause a significant overhead, but that's based on gut feeling not on hard data. I'll keep an eye on the queue times.

**Pull request merged**